packages/itwinui-css/src/utils/component-action/link-box.scss

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../mixins';
+
+@mixin iui-link-box {
+  position: relative;
+  width: fit-content;
+  height: fit-content;
+  cursor: pointer;
+
+  &:where(:focus-within) {
+    outline: var(--iui-size-3xs) solid var(--iui-color-border-accent);
+    border-radius: var(--iui-border-radius-1);
```

Comment:
border-radius doesn't belong here, because similar to `width`, this class should have no opinion on styling.

Reply:
Removed border radius and width.